.chloggen/break-exporterqueue-factory.yaml

```diff
@@ -0,0 +1,25 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. otlpreceiver)
+component: exporterqueue
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Remove exporterqueue.Factory in favor of the NewQueue function, and merge configs for memory and persistent.
```

Comment:
So it means that custom queue implementations are not allowed anymore? I guess that's ok. But we should probably explicitly call this out here. Also, the `exporterqueue.Queue` interface and all related API can be hidden now as well, right?

Reply:
Correct we can hide all of that. I think if customer wants custom queue they can have their own exporter pipeline, since queue is such a core of the pipeline now.

Reply:
Updated the subtext.

Reply:
Sounds good to me